TerminalCommands.md

```diff
@@ -103,7 +103,7 @@ defaults write com.knollsoft.Rectangle todo -int 1
 
 ## Only allow drag-to-snap when modifier keys are pressed
 
-Modifier key values can be ORed together.
```

Comment:
"ORed" is the correct term here. We can replace it with something like "Modifier key values can be combined by OR operation"